themes/root/js/captcha-altcha.js

```diff
@@ -0,0 +1,82 @@
+/* global VuFind */
+
+VuFind.register('altcha', function AltchaCaptchas() {
+  const READY_CLASS = "js-altcha-ready";
+  const CHALLENGE_SELECTOR = ".altcha-challenge";
+
+  let altchaLoaded = null;
+
+  /**
+   * @param {HTMLFormElement} form Container form
+   * @returns {void}
+   */
+  function altchaPerformWork(form) {
+  	if (altchaLoaded === null) {
+  		altchaLoaded = import("./vendor/altcha.js");
+  	}
+
+    const statusEl = form.querySelector("altcha-status");
+    statusEl.textContent = "Altcha loading...";
+
+  	altchaLoaded.then(({ Altcha }) => {
+    	statusEl.textContent = "Altcha starting work.";
+
+    	const challenge = form.querySelector(CHALLENGE_SELECTOR).value;
+  	});
```

Comment:
Weird indentation here -- I think you have a mix of tabs and spaces. I wonder if there's an eslint rule that could help with this.